tests/buffer/buffer_api_common.h

```diff
@@ -480,24 +481,27 @@ class check_buffer_linearization {
  public:
   void operator()(util::logger &log) {
     constexpr int size = 8;
-    // clang-format off
-    sycl::nd_range<1> range1d(sycl::range<1>{size},
-                              sycl::range<1>{size});
-    sycl::nd_range<2> range2d(sycl::range<2>{size, size},
-                              sycl::range<2>{size, size});
-    sycl::nd_range<3> range3d(sycl::range<3>{size, size, size},
-                              sycl::range<3>{size, size, size});
-    // clang-format on
+    auto q = util::get_cts_object::queue();
+    sycl::range<1> range1d = sycl_cts::util::work_group_range<1>(q);
+    sycl::range<2> range2d = sycl_cts::util::work_group_range<2>(q);
+    sycl::range<3> range3d = sycl_cts::util::work_group_range<3>(q);
```

Comment:
```suggestion
    sycl::range<1> range1d = sycl_cts::util::work_group_range<1>(q, size);
    sycl::range<2> range2d = sycl_cts::util::work_group_range<2>(q, size * size);
    sycl::range<3> range3d = sycl_cts::util::work_group_range<3>(q, size * size * size);
```
We don't need to use `max_work_item_sizes` to test buffer linearization functionality.

Reply:
FIxed it. Thanks!